packages/flame_riverpod/lib/flame_riverpod.dart

```diff
@@ -0,0 +1,7 @@
+/// Helpers for using Riverpod in conjunction with Flame, to share state from
+/// the game into other parts of your application, or from other parts of your
+/// application into your game.
```

Comment:
do we have such library/package-level comments elsewhere? is this supported by dartdoc?

Reply:
Yes (although it is not regularly used), and yes

Comments in this part of a library are shown underneath the name of the library in the 'Libraries' list of the generated docs. It is currently used elsewhere in:

- experimental in https://pub.dev/documentation/flame/latest/
- flame_lint https://pub.dev/documentation/flame_lint/latest/